src/main/java/seedu/address/logic/parser/AddressBookParser.java

```diff
@@ -59,6 +60,9 @@ public Command parseCommand(String userInput) throws ParseException {
         case AddCommand.COMMAND_WORD:
             return new AddCommandParser().parse(arguments);
 
+        case AddWorkExperienceCommand.COMMAND_WORD:
+            return new AddWorkExperienceCommandParser().parse(arguments);
```

Comment:
Add in a test case that checks whether the parser can detect the new command word added